docs/query-accelerator.md

```diff
@@ -28,12 +28,13 @@ be possible to replace a query set with a new version or to remove it completely
 properties or aspects applied to nodes and if necessary (for selected databases) the order of columns
 in compound indexes. Query sets are defined using JSON files.
 
-3. The addition of new query sets, the replacement of an existing query set or complete removal, require the
-Administrator to perform a query set refresh in the Alfresco Administration Console.
-The alfresco.log will contain messages to reflect progress. When a new query set is identified, the system will start populating a denormalized 
+3. The addition of new query sets, the replacement of an existing query set or complete removal does not require a 
+restart, an outage or have a major impact on normal operations. The alfresco.log will contain messages to reflect 
```

Comment:
I think this is not clear. We need to make sure that the admin performs the refresh action rather than thinks the system detects the changes automatically.

Reply:
Done